src/main/java/org/folio/rest/converter/packages/PackageCollectionResultConverter.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.rest.converter.packages;
+
+import static org.folio.common.ListUtils.mapItems;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.folio.holdingsiq.model.PackageData;
+import org.folio.holdingsiq.model.PackageId;
+import org.folio.holdingsiq.model.Packages;
+import org.folio.rest.jaxrs.model.MetaTotalResults;
+import org.folio.rest.jaxrs.model.PackageCollection;
+import org.folio.rest.jaxrs.model.PackageCollectionItem;
+import org.folio.rest.jaxrs.model.Tags;
+import org.folio.rest.util.RestConstants;
+import org.folio.rmapi.result.PackageCollectionResult;
+import org.folio.tag.repository.packages.DbPackage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PackageCollectionResultConverter implements Converter<PackageCollectionResult, PackageCollection> {
+
+  @Autowired
+  private Converter<PackageData, PackageCollectionItem> packageCollectionItemConverter;
+
+  @Override
+  public PackageCollection convert(@NonNull PackageCollectionResult packagesResult) {
+    Packages packages = packagesResult.getPackages();
+    List<DbPackage> dbPackages = packagesResult.getDbPackages();
+    List<PackageCollectionItem> packageList = mapItems(packages.getPackagesList(),
+      packageData -> {
+        PackageCollectionItem item = packageCollectionItemConverter.convert(packageData);
+        item.getAttributes()
+          .withTags(new Tags()
+            .withTagList(getTagsById(dbPackages, createPackageId(packageData))));
+        return item;
```

Comment:
@andrii-paias -- will tags now get included for all package collections in general? Realize the JIra issue was specific for provider (search packages within) -- but will tags also be calculated/returned for a general package list as well

Reply:
@carolegodfrey, 
no, they will only be returned if endpoint returns PackageCollectionResult (class with tag information), so now only packages withing providers have tags returned